pkg/traceql/engine.go

```diff
@@ -181,6 +181,9 @@ func (e *Engine) ExecuteTagValues(
 	// I.e. we are autocompleting resource.service.name and the query was {resource.service.name="foo"}
 	for _, c := range autocompleteReq.Conditions {
 		if c.Attribute == tag && c.Op == OpEqual {
+			// If the tag is already filtered in the query, then we can just return the operand
+			// as the only value.
+			cb(c.Operands[0])
```

Comment:
Operands is always guaranteed to have a length?

Reply:
No, it could be an empty string. I'll cover that case

Reply:
oh, sorry. i mean is the c.Operands slice always guaranteed to have something at index 0?

Reply:
IMO it needs both checks

Reply:
Actually, I don't see a good logic to skip values like empty string.

I've done a check for non-empty `c.Operands` only